lighthouse-cli/bin.js

```diff
@@ -102,21 +102,6 @@ async function begin() {
         ' Please use "--emulated-form-factor=none" instead.');
   }
 
-  if (typeof cliFlags.extraHeaders === 'string') {
```

Comment:
the `coerce()` functions from yargs gave a perfect place to do this loading/parsing and avoid the awkward type dance. Just moved into `cli-flags`